erpnext/public/js/telephony/telephony_console.html

```diff
@@ -0,0 +1,73 @@
+<div id="cc_console">
+	<div class="col-xs-4" style="padding:0px;">
+		<ul class="list-group">
+			{% if(info.title == "New Caller") { %}
+			<li class="list-group-item">
+			<span class="h3"><i class="fa fa-user" style="margin-right: 20px"></i>
+				New Caller
+			</span>
+			</li>
+			<li class="list-group-item">
+				<span class="h3"><i class="fa fa-phone" style="margin-right: 20px"></i>{{ info.number }}</span>
+			</li>
+			<li class="list-group-item">
+				<button id="new_lead" class="btn btn-sm btn-primary">Create Lead</button>
```

Comment:
public facing strings must be translatable

https://github.com/frappe/erpnext/wiki/Coding-Standards#user-facing-messages-must-be-translated